frontend/src/services/fetch/fetchers/savedOpportunityFetcher.ts

```diff
@@ -69,3 +70,22 @@ export const getSavedOpportunity = async (
   );
   return savedOpportunity ?? null;
 };
+
+export const fetchSavedOpportunities = async (): Promise<
+  SavedOpportunity[]
+> => {
+  try {
+    const session = await getSession();
```

Comment:
It may be overkill, but since the session will only be available on the server side (until now it's only been used in route handlers) maybe we should put a "server only" on this file?

Reply:
Agree there's no big risk here since this would break if we tried to use it in the client, but potentially would add some dev time if someone didn't realize that. If we pop "server-only" in here, since it's a utilities file rather than anything react specific, I think we'd gain the ability to break faster and provide an annotation for devs so that the use case for the file is immediately clear. https://nextjs.org/docs/app/building-your-application/rendering/composition-patterns#keeping-server-only-code-out-of-the-client-environment

Not entirely necessary, but worth maybe discussing further and thinking about documenting as we make a decision one way or another about how we want to use these designations.

Reply:
I was conflating `server-only` and `use server`, the former poisoning clients from importing while the latter endorsing. I added `server-only`.